package.json

```diff
@@ -19,15 +19,15 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.3",
-    "ember-cli": "2.6.2",
+    "ember-cli": "2.7.0",
```

Comment:
@stefanpenner note that this change should also include the `rootURL` adjustment